From 459011b0d703b3bf8c0b333de2dc83cde4b9ae6e Mon Sep 17 00:00:00 2001
From: Oliver Brown <oliver.brown@nxp.com>
Date: Wed, 24 Oct 2018 12:47:08 -0500
Subject: [PATCH 376/706] MLK-20063 drm: imx: dp: Correct iMX8M PHY
 initialization

The DisplayPort PHY initialization does not need the additional
loop gain enabled.

Signed-off-by: Oliver Brown <oliver.brown@nxp.com>
(cherry picked from commit 37e7b8bee61a87a6c14a5c498a7ae10b00cfe14d)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 drivers/gpu/drm/imx/hdp/API_AFE_mcu2_dp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/gpu/drm/imx/hdp/API_AFE_mcu2_dp.c b/drivers/gpu/drm/imx/hdp/API_AFE_mcu2_dp.c
index d69ca85495a0..946aa3a3b60b 100644
--- a/drivers/gpu/drm/imx/hdp/API_AFE_mcu2_dp.c
+++ b/drivers/gpu/drm/imx/hdp/API_AFE_mcu2_dp.c
@@ -506,7 +506,7 @@ void afe_init_t28hpc(state_struct *state,
 	val = val | 0x0008;
 	Afe_write(state, PHY_PMA_CMN_CTRL1, val);
 
-	Afe_write(state, CMN_DIAG_PLL0_TEST_MODE, 0x0022);
+	Afe_write(state, CMN_DIAG_PLL0_TEST_MODE, 0x0020);
 	Afe_write(state, CMN_PSM_CLK_CTRL, 0x0016);
 
 	if (refclk == REFCLK_24MHZ) {
-- 
2.17.1

